cluster/addons/dashboard/dashboard-rbac.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: v1
```

Comment:
the service account belongs in the `dashboard-controller.yaml` file... it is required whether or not RBAC is used

Reply:
Sure. Done.